rust/src/rtp_parameters.rs

```diff
@@ -704,6 +704,7 @@ pub struct RtpParameters {
 }
 
 impl RtpParameters {
+    #[allow(dead_code)]
```

Comment:
This is not a solution. Please don't add such things without TODO explaining why you have added it, why it is fine and when you expect to remove it.

Reply:
(typo fixed BTE, comment edited and new commit pushed)